tools/slintpad/src/editor_widget.ts

```diff
@@ -65,7 +65,7 @@ export function initialize(): Promise<void> {
     });
 }
 
-const hello_world = `import { Button, VerticalBox } from "std-widgets.slint";
+const hello_world = `import { AboutSlint, Button, VerticalBox } from "std-widgets.slint";
```

Comment:
Ooops, our commits crossed. I just ran into the same and pushed to master. Sorry :)